templates/docs/base/code.md

```diff
@@ -69,27 +77,13 @@ View example of the code copyable pattern
 
 Please copy the entire JS in the example, for copy to clipboard functionality.
 
-### Numbered
-
-The code numbered pattern can be used when displaying large blocks of code to enable users to quickly reference a specific line.
-
-<div class="embedded-example"><a href="/docs/examples/patterns/code-numbered/" class="js-example">
-View example of the code numbered pattern
-</a></div>
-
-### Import
-
-To import just code copyable or numbered component into your project, copy either or both snippets below and include it in your main Sass file.
+### Deprecated numbered
```

Comment:
Keep the title, by put `<span class="p-label--deprecated">Deprecated</span>` below.

Reply:
Ah, you have the label. So keep the title